packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -385,7 +383,7 @@ contract PerpetualPositionManager is FundingRateApplier {
         returns (FixedPoint.Unsigned memory amountWithdrawn)
     {
         PositionData storage positionData = _getPositionData(msg.sender);
-        require(numTokens.isLessThanOrEqual(positionData.tokensOutstanding), "Invalid token amount");
```

Comment:
:(

Reply:
This one I removed for the same reason as the `require(collateralAmount.isGreaterThan(0), "Invalid collateral amount")` which is that these requires should get hit less frequently because honest acting users won't try to input 0 tokens or a number thats clearly over their balance/debt outstanding.